src/components/Modal/Modal.mdx

```diff
@@ -25,6 +25,7 @@ import Link from "../Link/Link";
 - {<Link href="#modaltrigger-component-props" target="_self">ModalTrigger Component Props</Link>}
 - {<Link href="#usemodal" target="_self">useModal</Link>}
 - {<Link href="#usemodal-component-props" target="_self">useModal Component Props</Link>}
+- {<Link href="#confirmation-variant" target="_self">Confirmation Modal Variant</Link>}
```

Comment:
I feel this functionality is significant and should be moved further up in the TOC.  What do you think?  What was your reasoning for putting it in this slot?

Reply:
Agreed but i wanted someone reading through to understand the difference between `ModalTrigger`/`useModal` before getting to the variant . Not super attached to this slot though

Reply:
Ok, understood.  Leave it as is.